pom.xml

```diff
@@ -169,9 +169,9 @@
             <version>1.3</version>
         </dependency>
         <dependency>
-            <groupId>com.madgag.spongycastle</groupId>
-            <artifactId>core</artifactId>
-            <version>1.58.0.0</version>
+            <groupId>org.bouncycastle</groupId>
+            <artifactId>bcprov-jdk15on</artifactId>
```

Comment:
This will cause problems with `jetifier` on Android, we should use `<artifactId>bcprov-jdk15to18</artifactId>` instead.
See https://github.com/bcgit/bc-java/issues/762.
I checked that `bcprov-jdk15to18` does work on an Android emulator.

Reply:
Thanks for noticing that! Done in https://github.com/ACINQ/bitcoin-lib/pull/48/commits/1369eeb67523cd6c935a159f3cf4423c598d7e6c